### PR TITLE
chore(authenticator): Re-theme

### PR DESCRIPTION
**Pull request by dnys1**
*Issue #, if available:*

*Description of changes:*
- Update theme to match latest Amplify UI specs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_authenticator/assets/Inter/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2020 The Inter Project Authors (https://github.com/rsms/inter)
```
Comment (haverchuck) [inline packages/amplify_authenticator/assets/Inter/OFL.txt:1]: nit: The license in their GH has this line: 
`"Inter" is trademark of Rasmus Andersson.`
Not sure if that's important?

**Review by haverchuck — APPROVED** [review]
Left one comment about the license; otherwise, I'm looking forward to seeing this!

File: packages/amplify_authenticator/assets/Inter/OFL.txt (hunk 1 above)
Comment (dnys1) [reply packages/amplify_authenticator/assets/Inter/OFL.txt:1]: It's the license file they provide

File: packages/amplify_authenticator/lib/amplify_authenticator.dart [hunk 2]
```diff
@@ -89,8 +89,8 @@ class Authenticator extends StatefulWidget {
     ConfirmSignInMFAForm? confirmSignInMFAForm,
     this.stringResolver = const AuthStringResolver(),
     required this.child,
-  })  : useAmplifyTheme = false,
-        super(key: key) {
+    this.useAmplifyTheme = true,
```
Comment (Jordan-Nelson) [inline packages/amplify_authenticator/lib/amplify_authenticator.dart:92]: IMO this should still default to false. The standard material theme is going to feel _much_ more familiar to customers than the amplify theme. Also, if customers drop this into an existing MaterialApp that has the `theme` attribute set, the authenticator will pick up their theme automatically if this is false, which would be pretty nice.

On a related note, the tabs on the authenticator don't look very good if the user sets this to false. 

Screen Shot 2021-11-11 at 12 39 56 PM

**Pull request merged**